mypy.ini

```diff
@@ -0,0 +1,8 @@
+[mypy]
+check_untyped_defs = True
+disallow_any_generics = True
```

Comment:
Is it wise to start with disallowing generics? Given some of our code is meant to become a library, I'm wondering if we'll be using generics to provide a sensible interface.

Reply:
Thanks for the review @sigmavirus24 . I'll change that.